src/components/EmbeddedActivity.tsx

```diff
@@ -87,7 +88,11 @@ export default function EmbeddedActivity({ participant, activity, name, onComple
   const activateEmbeddedActivity = async (activity) => {
     setActivityId(activity.id)
     setSaved(false)
-    setSettings({ ...settings, settings: activity.settings, configuration: { language: i18n.language } })
+    setSettings({
+      ...settings,
+      settings: activity.settings,
+      configuration: { language: i18n.language, participant_id: participant.id },
```

Comment:
Please remove `participant_id` from the configuration.

Reply:
@avaidyam 
As in Balloon Risk.pptx it suggests that, each day all participants get the same 15 balloons with the same breakpoints (to decrease variability between participants).
For this we are currently storing the breakpoint as local storage based on participant id.

Reply:
Because we do not need to handle the case of multiple participants using the same smartphone, you should be able to store it in `localStorage` **without** the Participant ID.

Reply:
Okay, we will remove the Participant ID from the settings.